<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphvizWidget</class>
 <widget class="QWidget" name="GraphvizWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="GraphvizView" name="GraphView"/>
   </item>
   <item>
    <widget class="QWidget" name="ToolsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="LayoutGroupBox">
        <property name="title">
         <string>Graph layout</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Direction</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="RankDirComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Top to Bottom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bottom to Top</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Left to Right</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Right to Left</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ViewGroupBox">
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="FitViewButton">
           <property name="text">
            <string>Fit view to full graph</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color : #909090;</string>
        </property>
        <property name="text">
         <string>Use Ctrl key + mouse wheel for zooming</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="ExportGroupBox">
        <property name="title">
         <string>Export</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="PNGExportButton">
           <property name="text">
            <string>Export as PNG...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SVGExportButton">
           <property name="text">
            <string>Export as SVG...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>LayoutGroupBox</zorder>
     <zorder>LayoutGroupBox</zorder>
     <zorder>ViewGroupBox</zorder>
     <zorder>verticalSpacer_2</zorder>
     <zorder>label</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>ExportGroupBox</zorder>
     <zorder>verticalSpacer_3</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphvizView</class>
   <extends>QGraphicsView</extends>
   <header>GraphvizView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
